static/js/components/QuickSearchBar.jsx

```diff
@@ -11,20 +11,10 @@ import CircularProgress from "@material-ui/core/CircularProgress";
 
 import { GET } from "../API";
 
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme) => ({
   root: {
-    background: "#9394c3",
-    "& label": {
-      color: "black",
-    },
-    "& label.Mui-focused": {
-      color: "white",
-    },
-  },
-  label: {
-    "& label": {
-      color: "white",
-    },
+    background: theme.palette.primary.light,
+    borderRadius: '4px',
```

Comment:
Getting rid of the other styling seems to have made it good for dark mode, but now I feel that the light mode version could use some love:
![Screenshot 2021-02-09 130454](https://user-images.githubusercontent.com/17696889/107408028-64b51000-6ad8-11eb-8313-1ea7dbbb822b.png)
Perhaps we can bring back the other styling rules about `label` and use `theme.palette.info.main` or something like that to get the search box to match the other sidebar links?

Reply:
Yep! I wasn't happy with any versions of this, but agreed, I'd like to get this closer to the color of the other menu items.